.jshintrc

```diff
@@ -18,10 +18,13 @@
   "predef": [
     "exports", 
     "require",
-    "process"
+    "process",
+    "expect"
   ],
   "esnext": true,
   "shadow": false,
   "supernew": false,
-  "strict": false
+  "strict": false,
+  "mocha": true,
```

Comment:
This makes JSHint not scream at me for using Jest's global functions like `describe` and `it`